apis/swagger.yml

```diff
@@ -811,6 +811,34 @@ paths:
         - $ref: "#/parameters/id"
       tags: ["Network"]
 
+  /networks:
+      get:
+        summary: "List networks"
+        operationId: "NetworkList"
+        produces: ["application/json"]
+        responses:
+          200:
+            description: "Summary networks that matches the query"
+            schema:
+              type: "array"
+              items:
+                $ref: "#/definitions/NetworkInfo"
+          500:
+            $ref: "#/responses/500ErrorResponse"
+        parameters:
+          - name: "filters"
+            in: "query"
+            description: |
+              JSON encoded value of the filters (a `map[string][]string`) to process on the networks list. Available filters:
+                - `driver=<driver-name>` Matches a network's driver.
+                - `id=<network-id>` Matches all or part of a network ID.
+                - `label=<key>` or `label=<key>=<value>` of a network label.
+                - `name=<network-name>` Matches all or part of a network name.
+                - `scope=["swarm"|"global"|"local"]` Filters networks by scope (`swarm`, `global`, or `local`).
```

Comment:
Please remove all these filter things, since we do not use them in a quite long time.